Demo/Common/IntegrationExample.swift

```diff
@@ -77,12 +83,11 @@ internal final class IntegrationExample: APIClientAware {
     }
 
     internal func finish(with error: Error) {
-        let isCancelled = (error as? ComponentError) == .cancelled
```

Comment:
no need to cancel?

Reply:
yeah, I though why not show the alert when user cancels, I can revert if its needed

Reply:
👍